src/components/PeopleItem/PeopleItem.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useContext } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink';
+import { PeopleContext } from '../../PeopleContext';
+
+interface Props {
+  person: Person
+}
+
+export const PeopleItem: React.FC<Props> = ({ person }) => {
+  const {
+    motherName, sex, fatherName, slug, born, died,
+  } = person;
+
+  const { personSlug } = useParams();
+  const { persons } = useContext(PeopleContext);
+
+  const findPersonByName = (name: string): Person | undefined => {
+    return persons.find((p) => p.name === name);
+  };
+
+  const mother = findPersonByName(motherName ?? '');
+  const father = findPersonByName(fatherName ?? '');
+
+  const renderParentLink = (parent: Person | undefined, parentName: string) => {
+    return parent !== undefined ? <PersonLink person={parent} /> : parentName;
+  };
+
+  return (
+    <tr
+      key={slug}
+      data-cy="person"
+      className={cn({
+        'has-background-warning': slug === personSlug,
+      })}
+    >
+      <td>
+        <Link
+          to={`/people/${slug}`}
+          className={cn({ 'has-text-danger': sex === 'f' })}
+        >
+          {person.name}
+        </Link>
+      </td>
+
+      <td>{sex}</td>
+      <td>{born}</td>
+      <td>{died}</td>
+
+      <td>{motherName && renderParentLink(mother, motherName)}</td>
+      <td>{fatherName && renderParentLink(father, fatherName)}</td>
+    </tr>
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable